### ISSUE TITLE
Placeholder for tags is truncated when starting to type

### ISSUE THREAD

**Issue by imarklee** [joomla-projects/j4adminui#106]
### Steps to reproduce the issue
Create a new article and insert your first tag. Notice that the placeholder is being truncated.

### Expected result
The full placeholder to be visible.

### Actual result
Placeholder is being truncated.

<img width="934" alt="tags_placeholder_truncated" src="https://user-images.githubusercontent.com/892204/69546772-53fda480-0fcf-11ea-86e6-502318df1c8e.png">

**Comment by brianteeman** [comment]
Fairly certain that this is because this repo is running an outdated version of the choices.js library

**Comment by imarklee** [comment]
Ah I see. Should this repository be updated so that we can eliminate these issues?

**Comment by brianteeman** [comment]
Of course. It should always be in sync but its not going to be easy because of the technical decision by @marcodings to allow this repo to break everything even if its not template related until a later date. So yes - you may well end up wasting your time.

**Comment by imarklee** [comment]
@marcodings Is it possible to update the repository so that more tests can be done to make this template more stable for others to test as well?

Because it would be difficult to test not knowing if the problem is coming from the template or Joomla 4 itself.

**Comment by astridx** [comment]
> @brianteeman Of course. It should always be in sync but its not going to be easy because of the technical decision by @marcodings to allow this repo to break everything even if its not template related until a later date. So yes - you may well end up wasting your time.

I like that you explain the solution so often. But it is not a waste of time. In everything you do, you learn something and can use it in the future. Maybe there are so many problems with the template, because the rule is too strict and this way is good. We will see.

**Comment by marcodings** [comment]
> 
> 
> @marcodings Is it possible to update the repository so that more tests can be done to make this template more stable for others to test as well?
> 
> Because it would be difficult to test not knowing if the problem is coming from the template or Joomla 4 itself.

**Comment by marcodings** [comment]
Ooops, inadvertendly closed ..

> @marcodings Is it possible to update the repository so that more tests can be done to make this template more stable for others to test as well?

Yes, the developers @kawshar should rebase from time to time to "carry over" items fixed in the main branch

**Comment by marcodings** [comment]
> Of course. It should always be in sync but its not going to be easy because of the technical decision by @marcodings to allow this repo to break everything even if its not template related until a later date. So yes - you may well end up wasting your time.

Could you please stop rehashing this issue and please state facts as they were presented to you. The way of working is a decision by production not any individual
